python2/httplib2/socks.py

```diff
@@ -365,9 +366,17 @@ def __negotiatehttp(self, destaddr, destport):
         else:
             addr = destaddr
         headers =  ["CONNECT ", addr, ":", str(destport), " HTTP/1.1\r\n"]
-        headers += ["Host: ", destaddr, "\r\n"]
-        if (self.__proxy[4] != None and self.__proxy[5] != None):
-                headers += [self.__getauthheader(), "\r\n"]
+        didhost = False
```

Comment:
For readability, can you use different variable names? `didhost` and `didauth` are not very clear to me.